batch.py

```diff
@@ -488,11 +490,19 @@ def summarise_association_results(
     from multipy.fdr import qvalue
```

Comment:
will 
```from google.cloud import storage```
need to be re-imported inside this method call?

Reply:
No, should connect fine, but as you've done, you'll want to create the client inside this function